### DISCUSSION TITLE
No image files match: MD_skeletonized_masked

### DISCUSSION THREAD

**Discussion by cassiopeia199854** [isdneuroimaging/psmd#11] [Q&A]
When I run histogram analysis, why do I keep getting these lines:
...Histogram analysis
Image Exception : #63 :: No image files match: MD_skeletonized_masked
Image Exception : #22 :: Failed to read volume MD_skeletonized_masked.nii.gz
Error : No image files match: MD_skeletonized_masked
Failed to read volume MD_skeletonized_masked.nii.gz
Error : No image files match: MD_skeletonized_masked
Image Exception : #63 :: No image files match: MD_skeletonized_masked
Image Exception : #22 :: Failed to read volume MD_skeletonized_masked.nii.gz
Error : No image files match: MD_skeletonized_masked
Failed to read volume MD_skeletonized_masked.nii.gz
Error : No image files match: MD_skeletonized_masked
awk: syntax error at source line 1

I tried to change the name of the skeleton mask, but it can't work.

**Comment by marcoduering** [comment]
Sorry for not responding earlier, I totally missed your post. Please run the PSMD pipeline with the `-v` (verbose) option to get more detailed log output for debugging.
